assets/map/images/index.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
Adding type declaration for png files should help :) (`declarations.d.ts`)
I would try
```
declare module '*.png' {
  import { ImageSourcePropType } from 'react-native'

  const value: ImageSourcePropType;
  export default value;
}
```
https://stackoverflow.com/questions/51100401/typescript-image-import